openwisp_monitoring/monitoring/tests/__init__.py

```diff
@@ -7,13 +7,125 @@
 
 from ...db import timeseries_db
 from ...db.backends import TIMESERIES_DB
+from .. import register_chart, unregister_chart
 
 start_time = now()
 ten_minutes_ago = start_time - timedelta(minutes=10)
 Chart = load_model('monitoring', 'Chart')
 Metric = load_model('monitoring', 'Metric')
 AlertSettings = load_model('monitoring', 'AlertSettings')
 
+# chart configuration queries of InfluxDB for automated tests
+test_query = {
```

Comment:
I don't understand why these two data structures are being separated.
I think this makes my life harder because if I want to add or edit something I have to do it in two places.
Please merge this.

Reply:
PS: also rebase on the current dev and fix conflicts